src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -125,4 +125,8 @@ public void linkOnCommentsLink() {
         AppPage.getNewPageInstance(browser, StudentCommentsPage.class);
         
     }
+    
+    public void waitForCellHoverToDisappear() {
+        waitForElementToDisappear(By.className("cell-hover"));
```

Comment:
I cannot replicate the test failure right now though, so I cannot be 100% sure this does not work.
It seems that selenium may randomly hover over the elements triggering the failure?
One possible solution I can think of is to forcefully remove the cell-hover classes using selenium or JavascriptExecutor, since we are not testing the hover behaviors. Might not be a good idea though...
@kanghj What do you suggest?